src/calculateEntry.js

```diff
@@ -1,11 +1,26 @@
-const data = require('../data/zoo_data');
+const { prices } = require('../data/zoo_data');
 
-function countEntrants(entrants) {
-  // seu código aqui
-}
+const countEntrants = (entrants) => {
+  if (entrants === undefined || Object.keys(entrants).length === 0) return 0;
+
+  const entries = {
+    child: entrants.filter((child) => child.age < 18).length,
```

Comment:
```suggestion
    child: entrants.filter((entrant) => entrant.age < 18).length,
```

Pra mantermos a coerência e semântica, acho que faz sentido esse parametro se chamar `entrant`, o que acha?

Reply:
Caso concorde, se aplica às próximas linhas.

Reply:
Faz sentido! Vou alterar as 3! 😄 
Obrigado!